addons/website/models/ir_ui_view.py

```diff
@@ -140,46 +134,40 @@ def render(self, cr, uid, id_or_xml_id, values=None, engine='ir.qweb', context=N
 
             values = qcontext
 
-        return super(view, self).render(cr, uid, id_or_xml_id, values=values, engine=engine, context=context)
-
-    def _prepare_qcontext(self, cr, uid, context=None):
-        if not context:
-            context = {}
+        return super(View, self).render(cr, uid, id_or_xml_id, values=values, engine=engine, context=context)
 
-        company = self.pool['res.company'].browse(cr, SUPERUSER_ID, request.website.company_id.id, context=context)
+    @api.model
+    def _prepare_qcontext(self):
+        company = self.env['res.company'].sudo().browse(request.website.company_id.id)
```

Comment:
ok ok... if you really need superuser, you can do something like self.sudo().request.website.company_id :)